src/models/gaze16/internal-overview.md

```diff
@@ -12,7 +12,9 @@
 - M.2 SSDs are highlighted in yellow
 - 2.5" storage drive is highlighted in purple
 - CMOS battery connector is highlighted in cyan
-- BIOS flash chip (U66) is highlighted in black
+- BIOS flash chip is highlighted in black
+    - Labeled U66 on 3050/3050 Ti
```

Comment:
U51 on 3050

Reply:
That must have slipped through since I confirmed both of these numbers in Mattermost back in July. Corrected now.